log/logging.go

```diff
@@ -214,12 +225,14 @@ type Logger interface {
 	SetOutput(w io.Writer)
 	SetTimestampFlags(flags int)
 	SetStaticField(name string, value interface{})
+	SetStackTrace(trace bool)
 }
 
 // Logger config. Default/unset values for each attribute are safe.
 type Config struct {
 	Format LogFormat
 	ID     string
+	Depth  int
```

Comment:
No reason to expose this, right?

Reply:
I suppose not there is a Setter methods already